src/theme/input.ts

```diff
@@ -17,40 +17,40 @@ export default (options: Required<ModuleOptions>) => ({
     size: {
       xs: {
         base: 'px-2 py-1 text-xs gap-1',
-        leading: 'pl-2',
-        trailing: 'pr-2',
+        leading: '!ps-2',
```

Comment:
Why add `!` important class here?

Reply:
https://github.com/user-attachments/assets/c6000177-7148-4446-bd36-8288fe79dd1a

Reply:
https://github.com/user-attachments/assets/9aabe3db-da9d-4e4f-aa66-08577bd7bfd5

Reply:
@benjamincanac 
in version 4.0.0-alpha.28, the  `ps-10` is not applied as expected because `px-5` takes precedence, while in version 3.4.13, `ps-10` works correctly.

Reply:
This is an issue with Tailwind then, can't merge this with important values it will prevent override.

Reply:
@benjamincanac 
https://github.com/tailwindlabs/tailwindcss/pull/14777 😊😊

Reply:
Awesome! Let's wait for the new release then :) Thanks for taking care of the issue on tailwindcss!